src/en/credentials.md

```diff
@@ -213,20 +207,48 @@ juju add-credential azure -f mycreds.yaml
     
 ## Managing credentials
 
-There are several management tasks that can be done related to credentials.
+Here we cover the management of credentials.
 
-### Listing credentials
+ - [Setting default credentials][#setting-default-credentials]
```

Comment:
At this stage, I think that this command only operates on local credentials. It's in the works to ensure that users can set default on the remote ones too..